thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/detector/function/UserRuleAnomalyFunction.java

```diff
@@ -46,12 +46,6 @@
   public static final String DEFAULT_MERGED_MESSAGE_TEMPLATE =
       "threshold=%s, %s values: %s (%s / %s)";
   private static final Joiner CSV = Joiner.on(",");
-  private static final Joiner ANOMALY_RESULT_MESSAGE_JOINER = Joiner.on("...");
-  private static final NumberFormat PERCENT_FORMATTER = NumberFormat.getPercentInstance();
```

Comment:
keep the static instance +config rather than instantiating in getMergedAnomalyResultMessage each time (line 54 in the PR, 60 in master branch)